overview/dai.md

```diff
@@ -0,0 +1,58 @@
+
+Dai achieves stability through a combination of external market forces, economic incentive mechanisms, and policy tools controlled by MKR token holders. Many different market actors behaving in a self-interested manner contribute to its stability.
+
+There are four primary mechanisms Dai uses to remain stable.
+
+### Peg Stability Modules
+
+The Maker Protocol uses contracts called PSMs (Peg Stability Modules) that allow Dai to be directly redeemed for other cryptocurrencies pegged to the U.S. dollar.
+
+At any time, any user can receive minted Dai in exchange for these cryptocurrencies. If a Peg Stability Module contains sufficient stock, any user can destroy Dai in exchange for receiving these cryptocurrencies.
+
+This mechanism is the most effective at maintaining the Dai peg in the short term.
+
+For more information about the PSM, please refer to the [Maker Manual](https://manual.makerdao.com/module-index/module-psm).
+
+### Maker Vaults
+
+Maker Vaults have a crucial role as they enable the generation of Dai by locking up collateral. This process significantly affects the total Dai supply and ensures transparency and auditability. Vaults maintain overcollateralization using a Liquidation Ratio to protect Dai's backing from volatile collateral devaluation. Additionally, a debt ceiling is enforced to limit exposure to risky or illiquid collaterals. Stability Fees are imposed on Dai generators to effectively manage the peg, fund Maker Protocol development and maintenance, and incentivize active participation in governance.
+
+
+### Vaults Stability Fees
+
+When creating Dai using Maker Vaults, the vault owner pays an ongoing Stability Fee. Maker Governance can increase or decrease these fees. These fee adjustments can be used to encourage owners of Maker Vaults to create or destroy Dai due to an increased or decreased ongoing cost. In this way, Maker Governance can affect the _supply_ of Dai.
+
+This mechanism is most effective at maintaining the Dai peg in the medium/long term.
+
+###  The Dai Savings Rate
+
+The Dai Savings Rate (DSR) serves as an interest rate offered to Dai holders who choose to deposit their Dai into the DSR smart contract. This incentivizes users to retain their Dai within the system, as they earn rewards for doing so. As a dynamic parameter subject to governance by MKR token holders, the DSR can be adjusted based on market conditions and protocol requirements. This mechanism allows to manage the demand for Dai and contributes to the overall stability and resilience of the MakerDAO platform.
+
+This mechanism is most effective at maintaining the Dai peg in the medium/long term.
+
+### Global Settlement
+
+MKR Holders can decide to shut down the Maker Protocol in order to protect its users, this is known as Global Settlement. At Global Settlement, Dai can be redeemed directly for $1 of collateral that backs it.
```

Comment:
This is slightly misleading. After Global Settlement e.g. through ESM, Dai holders are entitled to a pro-rata share of the collateral after vault owners have withdrawn excess collateral. It may not be worth $1. Maybe have a look here: https://manual.makerdao.com/module-index/module-emergency-shutdown and then rewrite this part?